### PR TITLE
Add ci-tests for 8.6 directory

### PR DESCRIPTION
**Pull request by JacobiCamunda**
Closes https://github.com/camunda/web-modeler/issues/9822

With https://github.com/camunda/web-modeler/issues/9437 I introduced a new folder `docker-compose/camunda-8.6` which is not part of the ci test pipeline. With this PR I extended the pipeline to also run tests in the directory.

### CODE REVIEW

**Comment by JacobiCamunda** [comment]
@aabouzaid, could you please also take a look at the [failing health check](https://github.com/camunda/camunda-platform/actions/runs/9513104344/job/26222485873?pr=688) of elasticsearch. It happens currently only on the `docker-compose.yaml` test for now (yesterday on the modeler test) and only in the `8.6` directory. I don't know why that happens. Seems a bit flaky.

**Comment by aabouzaid** [comment]
@jessesimpson36 do you have any input about the E2E tests and ES health check?

File: .github/workflows/test-modeler.yaml [hunk 1]
```diff
@@ -5,8 +5,19 @@ on:
...
 jobs:
   test_modeler_yaml:
+    strategy:
+      matrix:
+        include:
+          - compose_args: "-f docker-compose.yaml -f docker-compose-web-modeler.yaml"
+            run_e2e_tests: true
+            directory: ""
+          - compose_args: "--profile modeling"
+            run_e2e_tests: false
```
[the remark this answers is not in the document]
Comment (jessesimpson36) [reply .github/workflows/test-modeler.yaml:15]: inside the `e2e_tests` folder, you can run `npx playwright test tests/web_modeler_login.spec.ts` if you just want to run the web modeler test.. that test will log into keycloak, put in a username/password of demo/demo, and verify that something is on the page after you log in.

If you don't want to run that test then leaving `run_e2e_tests: false` seems fine.

**Comment by jessesimpson36** [comment]
It looks to me like elasticsearch is unhealthy because it is in the yellow status, because it has unassigned shards from optimize.

```
curl localhost:9200/_cluster/health | jq         
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   394  100   394    0     0   219k      0 --:--:-- --:--:-- --:--:--  384k
{
  "cluster_name": "docker-cluster",
  "status": "yellow",
  "timed_out": false,
  "number_of_nodes": 1,
  "number_of_data_nodes": 1,
  "active_primary_shards": 66,
  "active_shards": 66,
  "relocating_shards": 0,
  "initializing_shards": 0,
  "unassigned_shards": 30,
  "delayed_unassigned_shards": 0,
  "number_of_pending_tasks": 0,
  "number_of_in_flight_fetch": 0,
  "task_max_waiting_in_queue_millis": 0,
  "active_shards_percent_as_number": 68.75
}

```
> curl localhost:9200/_cat/shards | grep UNASSIGNED
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100 11160    0 11160optimize-variable-label_v1                                    0 r UNASSIGNED                               
 optimize-process-definition_v6                                0 r UNASSIGNED                               
 optimize-instant-dashboard_v1                                 0 r UNASSIGNED                               
 optimize-business-key_v2                                      0 r UNASSIGNED                               
 optimize-tenant_v3                                            0 r UNASSIGNED                               
0optimize-decision-definition_v5                               0 r UNASSIGNED                               
 optimize-terminated-user-session_v3                           0 r UNASSIGNED                               
 optimize-import-index_v3                                      0 r UNASSIGNED                               
 optimize-license_v3                                           0 r UNASSIGNED                               
 optimize-process-overview_v2                                  0 r UNASSIGNED                               
 optimize-dashboard_v8                                         0 r UNASSIGNED                               
0optimize-collection_v5                                        0 r UNASSIGNED                               
 optimize-report-share_v3                                      0 r UNASSIGNED                               
 optimize-single-decision-report_v10                           0 r UNASSIGNED                               
1optimize-event-process-definition_v5                          0 r UNASSIGNED                               
2optimize-event-trace-state-external_v2                        0 r UNASSIGNED                               
2optimize-alert_v4                                             0 r UNASSIGNED                               
6optimize-event-sequence-count-external_v4                     0 r UNASSIGNED                               
koptimize-event-process-publish-state_v4                       0 r UNASSIGNED                               
 optimize-combined-report_v5                                   0 r UNASSIGNED                               
 optimize-variable-update-instance_v2-000001                   0 r UNASSIGNED                               
    0 --:--:-- --:--:-- --:--:-- 1362k
optimize-single-process-report_v11                            0 r UNASSIGNED                               
optimize-settings_v3                                          0 r UNASSIGNED                               
optimize-metadata_v3                                          0 r UNASSIGNED                               
optimize-external-process-variable_v2-000001                  0 r UNASSIGNED                               
optimize-position-based-import-index_v3                       0 r UNASSIGNED                               
optimize-timestamp-based-import-index_v5                      0 r UNASSIGNED                               
optimize-dashboard-share_v4                                   0 r UNASSIGNED                               
optimize-event_v4-000001                                      0 r UNASSIGNED                               
optimize-event-process-mapping_v4                             0 r UNASSIGNED
```

**Review by aabouzaid — CHANGES_REQUESTED** [review]
@JacobiCamunda could you please take a look at the comments from Jesse? :raised_hands:

**Comment by JacobiCamunda** [comment]
Hey, sorry that I only now came back to this. I had too much on my plate until now. I incorporated @jessesimpson36 proposal for the test and adjusted the test to just run the modeler test in this case. Also, I found out that the playwright modeler test doesn't work with the newest 8.6 alpha because Web Modeler removed the welcome part after logging in, which the test relies on. I adjusted the playwright test accordingly, too.

Successful test run: https://github.com/camunda/camunda-platform/actions/runs/10628330411

Unfortunately, I will be on FTO for three weeks starting next week. I will incorporate any change requests today and will answer questions still on Monday. Feel free to incorporate your feedback yourself in my PR if you can't provide feedback until EOB.

**Comment by jessesimpson36** [comment]
Why would operate and tasklist ping the healthcheck for zeebe to know if they're healthy?

**Review by jessesimpson36 — APPROVED** [review]

**Pull request merged**